qiita_db/user.py

```diff
@@ -349,55 +356,60 @@ def info(self, info):
 
     @property
     def default_analysis(self):
-        sql = ("SELECT analysis_id FROM qiita.analysis WHERE email = %s AND "
-               "dflt = true")
-        conn_handler = SQLConnectionHandler()
-        return conn_handler.execute_fetchone(sql, [self._id])[0]
-
-    @property
-    def sandbox_studies(self):
```

Comment:
It was used nowhere in the code, didn't even have a test in the user obj tests, and still references qiita.study_status table.